### PR TITLE
adding clean-up part

### PR DESCRIPTION
**Pull request by nguyt120**

### CODE REVIEW

File: fum_flows/models/integration/fum_flow.sql [hunk 1]
```diff
@@ -106,12 +105,11 @@ raw_deposit_account_customer AS (
         -- All transactions on month M reference M-1's MFI flag
         AND DATE_SUB(DATE_TRUNC(CURRENT_DATE(), MONTH), INTERVAL 1 day) = cmms.summary_date
     WHERE ctac.legal_owner_indicator = 'Y'
-    GROUP BY 1, 2, 3, 4, 5
+    GROUP BY 1, 2, 3, 4
```
Comment (khoinguyen-anz) [inline fum_flows/models/integration/fum_flow.sql:108]: Please specify on the columns name in GROUP BY

Comment (nguyt120) [reply fum_flows/models/integration/fum_flow.sql:108]: we would do this forr alll the files later, because queries might be changed

**Review by nguyea57 — APPROVED** [review]
LGTM

**Review by khoinguyen-anz — COMMENTED** [review]
LGTM

**Pull request merged**